packages/react/src/components/SplitButton/SplitButton.tsx

```diff
@@ -202,13 +205,13 @@ class SplitButton extends AutoControlledComponent<WithAsProp<SplitButtonProps>,
         )}
         {Button.create(toggleButton, {
           defaultProps: () => ({
+            styles: styles.toggleButton,
             className: SplitButton.slotClassNames.toggleButton,
             disabled,
-            icon: 'stardust-arrow-down',
+            icon: 'chevron-down-medium',
             iconOnly: true,
             primary,
             secondary,
-            ...accessibility.attributes.toggleButton,
```

Comment:
Removing this fixes some existing accessibility issues where the toggle button isn't focusable or clickable, but I'm not sure what this was doing...

Reply:
Easter egg :)

First of all the PR should fix styles, not accessibility. Secondly, the specs we had when developing the button was that the toggle button should not be tabbable by itself, only clickable. Just the main button should be tabbable, and the keyboard focus from that main button should apply a style on the whole split button (main + toggle).

@mnajdova Don't we have this covered in our isConformant tests? Or can we do it?

Reply:
What would we cover in the isConformant test regarding this? It seems to me that this should be covered in the unit tests for the component (if the behavior is changed).